packages/cashier/src/pages/payment-agent/payment-agent-details/payment-agent-details.tsx

```diff
@@ -1,75 +1,76 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Icon } from '@deriv/components';
 import './payment-agent-details.scss';
 
-const Detail = ({ action, icon, is_last_child, children, ...rest }) => {
-    const detail = Array.isArray(children) ? children : children.split(',');
+type TDetailsProps = {
```

Comment:
@hamid-deriv Is it the requirement to have all parameters optional? 🤔

In most of the added types everything has been marked as optional, By doing so we won’t really take advantage of typescript in our codebase.